github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml

```diff
@@ -420,6 +420,21 @@ presets:
   volumeMounts:
   - name: docker-graph
     mountPath: /docker-graph
+- labels:
+    preset-pinc-enabled: "true"
+  env:
+  - name: PODMAN_IN_CONTAINER_ENABLED
+    value: "true"
+  - name: KIND_EXPERIMENTAL_PROVIDER
+    value: "podman"
+  - name: PODMAN_IN_CONTAINER_IPV6_ENABLED
+    value: "true"
```

Comment:
question, do we really need all the parameters ?
what if we can have fewer global ones that if declared the others will be internally declared ?

Reply:
yes it might be better to set some of these in the runner.sh script instead.